packages/@react-spectrum/menu-trigger/src/MenuHeading.tsx

```diff
@@ -0,0 +1,21 @@
+import {classNames} from '@react-spectrum/utils';
+import {Heading} from '@react-spectrum/typography';
+import React from 'react';
+import {SpectrumMenuHeadingProps} from '@react-types/menu';
+import styles from '@adobe/spectrum-css-temp/components/menu/vars.css';
+
+export function MenuHeading<T>({item}: SpectrumMenuHeadingProps<T>) {
+  return (
+    <Heading
+      UNSAFE_className={
+        classNames(
+          styles, 
+          'spectrum-Menu-sectionHeading'
```

Comment:
I get having a component that for MenuHeading/MenuDivider, but should we use the slots API for this instead? we don't need to use grid, just slots to pass down these classNames